frontend/src/lib/derived/icrc-universes.derived.ts

```
@@ -23,12 +23,11 @@ const convertIcrcCanistersToUniverse = ({
   const universeId = canisters.ledgerCanisterId.toText();
   const token = tokensData[universeId];
   // TODO: Read logo from token https://dfinity.atlassian.net/browse/GIX-2140
-  // TODO: Add ckETH and ckETHTEST logos
   const logo =
     universeId === CKETH_UNIVERSE_CANISTER_ID.toText()
-      ? CKBTC_LOGO
+      ? CKETH_LOGO
       : universeId === CKETHSEPOLIA_UNIVERSE_CANISTER_ID.toText()
-      ? CKTESTBTC_LOGO
+      ? CKSEPOLIAETH_LOGO
       : undefined;
   if (isNullish(token) || isNullish(logo)) {
```

is it correct to exclude an universe correctly configured but for which only the logo would not be provided? UI we can fallback on a placeholder, we do that in Oisy Wallet or VRC.

---

Yes, that sounds good once we have a default logo.

But ideally we'll get the logo from the token metadata. But I guess we will still need the default logo.